### PR TITLE
[R2] Added Data Migration & R2 Migrator pages

### PR DESCRIPTION
**Pull request by dcpena**
Added docs for R2 Migrator to address PCX-5099.

### CODE REVIEW

File: content/r2/data-migration/r2-migrator.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: R2 Migrator
+pcx_content_type: how-to
+---
+
+# R2 Migrator
+
+{{<Aside type="note">}}
```
Comment (kodster28) [inline content/r2/data-migration/r2-migrator.md:10]: I'd suggest either moving this down below the initial sentence or manually specifying the meta description in the frontmatter.

Right now, the meta description is pulling in the aside.

File: content/r2/data-migration/r2-migrator.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: R2 Migrator
+pcx_content_type: how-to
+---
+
+# R2 Migrator
```
Comment (kodster28) [inline content/r2/data-migration/r2-migrator.md]: Beta tag?

File: content/r2/data-migration/r2-migrator.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                "arn:aws:s3:::<BUCKET_NAME>/*"
+            ]
+        }
+    ]
+}
+```
+
+3. Create a new user and attach the created policy to that user.
+
+You can now use both the Access Key ID and Secret Access Key when defining your source bucket. Refer to Migrating data from Amazon S3 to R2 to learn more.
```
Comment (kodster28) [inline content/r2/data-migration/r2-migrator.md]: Missing link for `Migrating data from Amazon S3...`?

File: content/r2/data-migration/r2-migrator.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+## Create Amazon S3 credentials
+
+To migrate objects from Amazon S3, R2 Migrator requires access permissions to your bucket. While you can use any AWS Identity and Access Management (IAM) user credentials with the correct permissions with the R2 Migrator, Cloudflare recommends you create a user with a narrow set of permissions.
+
+To create credentials with the correct permissions:
+
+1. Log in to your AWS IAM account.
+2. Create a policy with the following format and replace `<BUCKET_NAME>` with the bucket you want to grant access to:
+
+```js
```
Comment (kodster28) [inline content/r2/data-migration/r2-migrator.md]:
```suggestion
```json
```

**Review by kodster28 — APPROVED** [review]

File: content/r2/data-migration/r2-migrator.md (hunk 1 above)
Comment (dcpena) [reply content/r2/data-migration/r2-migrator.md:10]: Ah good callout. Thanks! Moved.

File: content/r2/data-migration/r2-migrator.md (hunk 3 above)
Comment (dcpena) [reply content/r2/data-migration/r2-migrator.md]: Added link

File: content/r2/data-migration/r2-migrator.md (hunk 4 above)
Comment (dcpena) [reply content/r2/data-migration/r2-migrator.md]: Fixed

File: content/r2/data-migration/r2-migrator.md (hunk 2 above)
Comment (dcpena) [reply content/r2/data-migration/r2-migrator.md]: Fixed

**Pull request merged**